plugins/org.locationtech.udig.libs/pom-libs.xml

```diff
@@ -74,6 +73,20 @@
 	</dependencyManagement>
 
 	<dependencies>
+		<!-- see org.locationtech.udig.libs.jfreechart -->
+		<dependency>
+			<groupId>jfree</groupId>
+			<artifactId>jfreechart</artifactId>
+			<version>1.0.13</version>

```

Comment:
You could have used the property instead of the duplicated version here, right? But anyway, can stay as is from my perspective :)

Reply:
created an issue for eastwood #562